klippy/toolhead.py

```diff
@@ -505,7 +505,7 @@ def get_status(self, eventtime):
                      'position': homing.Coord(*self.commanded_pos),
                      'max_velocity': self.max_velocity,
                      'max_accel': self.max_accel,
-                     'max_accel_to_decel': self.requested_accel_to_decel,
+                     'max_accel_to_decel': self.max_accel_to_decel,
```

Comment:
Thanks for the heads up.  Mainsail uses this data inform the UI of the existing velocity/accel settings.  I don't think changing it to the internal value would break things if you choose to change it.   Worst case scenario we can report both values as you suggest.  I will run it by Meteyou.

Reply:
In principle, the name of the exported variable does not change, but only the value does. As I mentioned in the comment below, this change actually 'fixes' the export to the actual value used by Klipper (i.e. if the user requested too high value and that value was 'trimmed' to the lower value). But I do not think this particular change is very important, I made it more for consistency, of reporting and to avoid the confusion, so I can revert it if you feel strongly against it. @Arksine in light of this, what do you think Mainsail should report?

Reply:
FWIW, I tend to think of "requested max accel to decel" to be the "real" value, and the internal "trimmed max accel to decel" to be an internal implementation detail.  That is, if the requested accel to decel is greater than the max accel, then it doesn't actually impact the print, but it's still a "valid value".

Looking closer, the new code doesn't actually call the `get_status()` method at all.  So, I think it would be better to separate this out into a separate PR (if needed at all).

Reply:
I moved these changes to a separate PR #3422, it is probably best to continue the discussion there. But TBH I do not have a very strong opinion about 'max accel to decel', I just made this change for consistency with the rest of reporting, and because the user may be more interested in the actually used value (even if it is an implementation detail, if a user requested to change max accel to decel, they probably want to know whether this request succeeded or not).